prov/efa/src/rdm/efa_rdm_pke_rtm.c

```diff
@@ -635,11 +651,8 @@ void efa_rdm_pke_handle_eager_rtm_send_completion(struct efa_rdm_pke *pkt_entry)
  */
 ssize_t efa_rdm_pke_proc_matched_eager_rtm(struct efa_rdm_pke *pkt_entry)
 {
-	int err;
-	int hdr_size;
-	struct efa_rdm_ope *rxe;
-
-	rxe = pkt_entry->ope;
+	int err = 0;
+	struct efa_rdm_ope *rxe = pkt_entry->ope;
 
 	if (pkt_entry->alloc_type != EFA_RDM_PKE_FROM_USER_BUFFER) {
```

Comment:
We shouldn't expect a user buffer allocated pkt if we handle zcpy recv completion outside `efa_rdm_pke_proc_received`